worlds/psychonauts/Options.py

```diff
@@ -0,0 +1,156 @@
+from dataclasses import dataclass
+from typing import List
+
+from Options import Choice, Range, Toggle, PerGameCommonOptions
+
+
+class StartingLevitation(Toggle):
+    """Start with Levitation Level 1"""
+    display_name = "Start with Levitation"
+    default = False
+
+
+class StartingMentalMagnet(Toggle):
+    """Start with Mental Magnet, pulling health, ammo, and arrowhead drops towards you."""
+    display_name = "Start with Mental Magnet"
+    default = True
+
+
+class StartingCobwebDuster(Toggle):
+    """Start with the Cobweb Duster."""
+    display_name = "Start with Cobweb Duster"
+    default = False
+
+
+class RandomStartingMinds(Range):
+    """Start with a random number of mind/area unlocking items."""
+    display_name = "Random Starting Minds"
+    range_start = 0
+    range_end = 10
+    default = 3
+
+
+class LootboxVaults(Toggle):
+    """Turns all Memory Vaults into Lootboxes! Vaults will contain 10-50 Arrowheads, with a 50% chance to recieve a rank up.
+    If you get really lucky, you can even win jackpots of up to 250 Arrowheads or Five Ranks Up!
+    If False, Vaults will always give One Rank Up and 15 Arrowheads."""
+    display_name = "Lootbox Vaults"
+    default = True
+
+class FasterLindaBoss(Toggle):
+    """Make the Linda Boss chase sequences faster by increasing the speed of the air bubble."""
+    display_name = "Faster Linda Boss"
+    default = True
+
+class EasyMillaRace(Toggle):
+    """Make the race in Milla's Mind easier by removing Bobby from the race, and increasing player speed to 1.5x"""
+    display_name = "Easy Milla Race"
+    default = False
+
+
+class EasyFlightMode(Toggle):
+    """Replicate the Levitation Flight Glitch by simply holding down float in the air. No more button mashing.
+    WARNING: Makes getting locations out of logic VERY easy."""
+    display_name = "Easy Flight Mode"
+    default = False
+
+
+class EnemyDamageMultiplier(Range):
+    """Adjust how much damage Raz takes from enemies for an additional challenge.
+    If set to 0, Raz will take no damage from enemies, but will still flinch."""
+    display_name = "Enemy Damage Multiplier"
+    range_start = 0
+    range_end = 5
+    default = 1
+
+
+class InstantDeathMode(Toggle):
+    """Take any amount of damage, die instantly. Has priority over Enemy Damage Multiplier."""
+    display_name = "Instant Death Mode"
+    default = False
+
+
+class Goal(Choice):
+    """Win Condition
+    Asylum Brain Tank: Climb the Asylum and defeat the Coach Oleander Brain Tank.
+
+    Brain Hunt: Find the required amount of Camper Brains and re-brain them in Ford's Sanctuary.
+
+    Asylum Brain Tank and Brain Hunt: Find the required amount of Camper Brains, AND defeat the Coach Oleander Brain Tank."""
+    display_name = "Goal"
+    option_braintank = 0
+    option_brainhunt = 1
+    option_braintank_and_brainhunt = 2
```

Comment:
I'm nitpicking, but I think it's a bit misleading that those are called `Asylum Brain Tank` and `Brain Hunt` in the doc, but the option to choose is `Braintank` and `Brainhunt`. It would be great to rename those into `option_asylum_brain_tank` so it's formatted like in the doc.

![image](https://github.com/ArchipelagoMW/Archipelago/assets/16137441/0a4db903-834f-430e-b097-975b817861bd)